webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleHandler.java

```diff
@@ -73,13 +72,11 @@ protected void wrappedService(HttpServletRequest request, HttpServletResponse re
 
                 sendBytes(request, response, totalPdfOutput, "application/pdf");
             } else if(ext.equals("zip")) {
-                ScrambleRequest[] scrambleRequests = ScrambleRequest.parseScrambleRequests(query, seed);
-                ByteArrayOutputStream baosZip = ScrambleRequest.requestsToZip(getServletContext(), globalTitle, generationDate, scrambleRequests, seed, generationUrl);
+                ByteArrayOutputStream baosZip = ScrambleRequest.requestsToZip(getServletContext(), globalTitle, generationDate, query);
```

Comment:
Stuff like `query` should not escape out of `*Handler` classes. It violates the abstraction we're trying to have between "this is our API", and "the internal business logic of tnoodle".

I preferred how this was before: this file would handle extracting/parsing stuff from the query, and pass the structured data `requestsToZip` needs to work.